src/posts/posts.controller.ts

```diff
@@ -66,10 +66,12 @@ export class PostsController {
 
   @ApiBearerAuth()
   @UseGuards(RequiredAuthGuard)
-  @Delete('/:postid')
-  async deletePost(@Param('postid') postid: string): Promise<string> {
+  @Delete('/:postId')
+  async deletePost(@Param('postId') postId: string): Promise<string> {
     // TODO
-    return `delete postid = ${postid}`;
+    return (await this.postsService.deletePost(postId))
```

Comment:
Since this is a REST API, best if all controllers always only return json objects and not bare strings anywhere

Reply:
Yeah, that totally makes sense. I've updated the `deletePost` controller. Let me know if you need any changes to that. :)

***After Update***

- [x] Returns true on successful delete
![delete-post-by-id](https://user-images.githubusercontent.com/39665013/128741486-72efb332-2aeb-43e1-8ba2-a34b19d053a8.png)

- [x] Return false if it's already deleted or if there's no post with that id
![cannot-delete-post](https://user-images.githubusercontent.com/39665013/128741534-db02f30f-d8c6-477f-a5cf-c207f25d96ee.png)